app/views/ViewUtils.scala

```diff
@@ -58,7 +58,7 @@ object ViewUtils {
           value = Some(a.value),
           checked = field.value.contains(a.value),
           content = Text(messages(a.messageKey)),
-          attributes = if (trackGa) Map[String, String]("data-journey-click" -> s"trusts-frontend:click:${a.id}") else Map.empty
```

Comment:
As the only usage of trackGa has been removed the parameter can be removed from the method signature and usages

Reply:
Done